catalog/pipeline/release/0.1/release.yaml

```diff
@@ -3,7 +3,13 @@ apiVersion: tekton.dev/v1beta1
 kind: Pipeline
 metadata:
   name: release
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
 spec:
+  description: >-
+    Tekton pipeline to release HACBS Application Snapshot to Quay.
```

Comment:
In their example, they say that descriptions should be a one line summary not ending on `.`. Could we remove those `.`?

Reply:
Yes we could remove them. :+1: 
The spec doesn't mention punctuation for line-ending, nor does the example show any.
HOWEVER, they do mention `git-clone` as their guiding example:
https://github.com/tektoncd/catalog/blob/main/task/git-clone/0.1/git-clone.yaml

Here is what `git-clone` does:
```
spec:
  description: >-
    These Tasks are Git tasks to work with repositories used by other tasks
    in your Pipeline.

    The git-clone Task will clone a repo from the provided url into the
    output Workspace. By default the repo will be cloned into the root of
    your Workspace. You can clone into a subdirectory by setting this Task's
    subdirectory param.

```
I've already removed them in the latest commit.
Thanks for the review!

Reply:
This link shows an example for short summaries https://github.com/tektoncd/community/blob/main/teps/0003-tekton-catalog-organization.md#example-of-a-resource-from-catalog. In there, there's no period.

Reply:
Agreed.

I took a quick look at the `git blame` for [git-clone](https://github.com/tektoncd/catalog/tree/main/task/git-clone) task, and the text taken for the example has never NOT had the punctuation. Honestly I guess somebody either accidentally omitted the punctuation on the subject line (copy pasta), or willfully excluded it. Either way, I tend to agree with you, the SPEC should prevail.

Here is the relevant upstream commit where it went from no punctuation, to being punctuated:
https://github.com/tektoncd/catalog/commit/8e1e334c92bf59532b701f081d1c9482c18040e2

Anyways, it's all for our amusement at this point. We're not going to have punctuation on the subject lines. :-)